substratools/utils.py

```diff
@@ -63,12 +63,23 @@ def load_interface_from_module(module_name, interface_class,
             # XXX don't use ModuleNotFoundError for python3.5 compatibility
             raise
 
-    # check interface
+    # find interface class
     for name, obj in inspect.getmembers(module):
         if inspect.isclass(obj) and issubclass(obj, interface_class):
             return obj()  # return interface instance
 
-    # backward compatibility; accep
+    # check module is not empty: this is done by checking that there is at least one
```

Comment:
I think it's better to check what you directly need in the module :
```python
>>> inspect.getmembers(toto, inspect.isclass)
[]
>>> inspect.getmembers(toto, inspect.isfunction)
[]
```

Reply:
👍

Reply:
done